lib/SnippetsTestMessageProvider.jsm

```diff
@@ -90,6 +90,25 @@ const MESSAGES = () => [
       block_button_text: "Block",
     },
   },
+  {
+    id: "SIMPLE_TEST_MONITOR",
+    template: "simple_snippet",
+    content: {
+      icon: TEST_ICON,
+      icon_dark_theme: TEST_ICON_BW,
+      button_label: "Get monitor",
+      button_action: "ENABLE_FIREFOX_MONITOR",
+      button_action_args: {
+        utm_term: "monitor-snippet-test",
```

Comment:
Not sure what the correct values for any of these should be; should they all just come from the snippets service?

Reply:
They're all strings so it seems ok.